src/scrapers/mizrahi.ts

```diff
@@ -130,6 +162,70 @@ function convertTransactions(txns: ScrapedTransaction[]): Transaction[] {
   });
 }
 
+async function getTransactionExtraScrap(record: ScrapedTransaction, headers: Headers): Promise<ExtraTransactionResult | null> {
+  const formattedPeulaDate = moment(record.MC02PeulaTaaEZ).format(DATE_FORMAT);
+  const data = {
+    inKodGorem: record.MC02KodGoremEZ,
+    inAsmachta: record.MC02AsmahtaMekoritEZ,
+    inSchum: record.MC02SchumEZ,
+    inNakvanit: record.MC02KodGoremEZ,
+    inSugTnua: record.MC02SugTnuaKaspitEZ,
+    inAgid: record.MC02AgidEZ,
+    inTarPeulaFormatted: formattedPeulaDate,
+    inTarErechFormatted: formattedPeulaDate,
+    inKodNose: record.MC02SeifMaralEZ,
+    inKodTatNose: record.MC02NoseMaralEZ,
+    inTransactionNumber: record.TransactionNumber,
+  };
+  
+  try {
+    const res = await fetchPost(TRANSACTION_DETAILS_REQUEST_URL, data, headers);
+    return res;
+  } catch (e) {
+    console.error(`Error fetching extra transaction details for record ${JSON.stringify(record)}`, e);
+  }
+  return null;
+}
+
+function simplifyExtraTransactionResultsToMemo(extraResult: ExtraTransactionResult): string {
+  let memo = '';
+  extraResult.body.fields.forEach(field => 
+    field?.forEach(group => 
+      group?.Records.forEach(record => 
+        record?.Fields.forEach((fieldRecord) => {
+          memo += `${fieldRecord.Label} ${fieldRecord.Value}; `;
+        }),
+      ),
+    ),
+  );
+  return memo;
+}
```

Comment:
I'm not sure we want to build a memo by ourself.

Maybe you can add an example of an object and we can learn what are you building here.